libbeat/outputs/kafka/client.go

```diff
@@ -76,7 +78,8 @@ func newKafkaClient(
 	index string,
 	key *fmtstr.EventFormatString,
 	topic outil.Selector,
-	writer codec.Codec,
+	headers map[string]string,
+    writer codec.Codec,
```

Comment:
could you format the whitespace, please?

Reply:
I have formatted this code, thanks !

Reply:
> could you format the whitespace, please?

I have formatted this code, thanks !